src/control.jl

```diff
@@ -59,7 +59,7 @@ end
 """
     FieldControlPoint(R::Real, Z::Real, θ::Real, target::Real, weight::Real=1.0)
 
-Return a control point for a `target` magnetic field `B(R, Z, θ) = BR(R, Z) * cos(θ) + BZ(R,Z) * sin(θ)` following the IMAS convention, with an optional `weight`
+Return a control point for a `target` magnetic field `B(R, Z, θ) = BR(R, Z) * cos(θ) - BZ(R,Z) * sin(θ)` following the IMAS convention, with an optional `weight`
```

Comment:
Define more what the IMAS convention is here and why it causes this minus sign, as it's unintuitive to me.